### PR TITLE
Added a few new features/fixed Sublist3r not installed bug

### PR DESCRIPTION
**Pull request by DKrypt1337**
* Redid the whole tool to have a new theme
* Part of the new theme highlights found domains/CVEs/etc in the list
* Fixed bug where Sublist3r "wasn't installed"
* I added a start, pause, and stop button (hidden for now).. In the future it can be used to start, pause, or stop the scan(s).
* Updated the domains to be placed into a dropdown box. So they're all in one spot, and not limited.

I'm going to do more testing to verify everything works as it should. Wanted to submit so others can help test to see if anything is broken as well.

### CODE REVIEW

**Comment by R-s0n** [comment]
This is amazing, thank you!!  It will be a bit before I get a chance to review and merge it, but I'll get to it ASAP!

File: install.py
Comment (R-s0n) [inline install.py:1]: What version of Kali are you using, currently?  The 2023 distro updated to a new version of Debian which requires you to use `apt` with `python3-` appended to the front of the package name for any Python installs.  Pip3 isn't supported anymore.

I did end up fixing the Sublist3r issue, though.  The original package isn't being maintained anymore but someone else is maintaining a fork that works great, so I just replaced the repo source.  Everything else in the tool is the exact same.

File: client/package.json
Comment (R-s0n) [inline client/package.json:1]: Where is `child_process` being used?  Am I missing an import somewhere?

File: install.py
Comment (DKrypt1337) [reply install.py:1]: I have Parrot OS, the newest version. Sounds like you got it fixed though :D

File: client/package.json
Comment (DKrypt1337) [reply client/package.json:1]: I believe it is used where the Sublist3r check was at. This was using Python to create a subprocess when calling Sublist3r I believe.

**Pull request closed without merging**